### PR TITLE
feat: add command to create and publish feature subgraph in one command

### PR DESCRIPTION
**Pull request by JivusAyrus**
This pr adds `wgc feature-subgraph publish` which publishes feature subgraphs if they exist and if they don't, creates them and then publishes them.

## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: cli/src/commands/subgraph/commands/create.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default (opts: BaseCommandOptions) => {
   );
   command.option('--readme <path-to-readme>', 'The markdown file which describes the subgraph.');
   command.option(
-    '-edg, --event-driven-graph',
+    '--edg, --event-driven-graph',
```
Comment (Aenimus) [inline cli/src/commands/subgraph/commands/create.ts:52]: Please make sure to update the docs.

File: controlplane/test/feature-subgraph/create-feature-subgraph.test.ts [hunk 2]
```diff
@@ -166,8 +169,73 @@ describe('Create feature subgraph tests', () => {
...
+
+    const featureSubgraphResponseOne = await client.publishFederatedSubgraph({
+      name: featureSubgraphName,
+      routingUrl: DEFAULT_SUBGRAPH_URL_TWO,
+      isFeatureSubgraph: true,
+      schema: 'type Query { hello: String }',
+    });
+    expect(featureSubgraphResponseOne.response?.code).toBe(EnumStatusCode.ERR_NOT_FOUND);
+    expect(featureSubgraphResponseOne.response?.details).toBe(
+      `Feature Subgraph ${featureSubgraphName} not found. If intended to create and publish, please pass the name of the base subgraph with --subgraph option.`,
```
[the remark this answers is not in the document]
Comment (JivusAyrus) [reply controlplane/test/feature-subgraph/create-feature-subgraph.test.ts:215]: Done

File: cli/src/commands/feature-subgraph/commands/publish.ts [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        }
+
+        break;
+      }
+      default: {
+        spinner.fail(`Failed to publish feature subgraph "${name}".`);
+        if (resp.response?.details) {
+          console.error(pc.red(pc.bold(resp.response?.details)));
+        }
+        process.exit(1);
```
[the remark this answers is not in the document]
Comment (JivusAyrus) [reply cli/src/commands/feature-subgraph/commands/publish.ts:193]: Done

File: cli/src/commands/feature-subgraph/index.ts [hunk 4]
```diff
@@ -2,12 +2,14 @@ import { Command } from 'commander';
 import { BaseCommandOptions } from '../../core/types/types.js';
 import { checkAuth } from '../auth/utils.js';
 import CreateFeatureSubgraphCommand from './commands/create.js';
+import PublishFeatureSubgraphCommand from './commands/publish.js';
 
 export default (opts: BaseCommandOptions) => {
   const command = new Command('feature-subgraph').alias('fs');
   command.description('Provides commands for creating and managing feature subgraphs.');
```
Comment (StarpTech) [inline cli/src/commands/feature-subgraph/index.ts:9]: Can we add here a small definition of feature subgraphs?

Comment (JivusAyrus) [reply cli/src/commands/feature-subgraph/index.ts]: Done

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**